web-app/src/shared/components/datetime-widget.component.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="m-4 space-y-3">
+    <p class="text-4xl">today</p>
+    <p>{{ formatDate(now) }}</p>
+  </div>
+</template>
+
+<script setup lang="ts">
+const now = new Date();
+
+const formatDate = (date: Date): string => {
+  const days = ["mon", "tue", "wen", "thu", "fri", "sat", "sun"];
+  return (
+    days[date.getDay()] +
+    " " +
+    date.toLocaleDateString("en-US") +
+    " | " +
+    date.toLocaleTimeString("en-US", { timeStyle: "short" })
+  );
```

Comment:
Leverage the [built-in apis](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat)